### ISSUE TITLE
[Bug]: Error

### ISSUE THREAD

**Issue by cands** [chrisgrieser/alfred-read-later#1]
### Bug Description

Great idea for a useful workflow! Unfortunately I get an error when running the workflow, in the Alfred debugger it is stated that "execution error: Error: Error: exception raised by object: *** +[NSString stringWithUTF8String:]: NULL cString (-2700)"

I'm on macOS 13.6 and using Alfred 5.1.4

### To Reproduce

Just run the workflow with a browser frontmost (tested Firefox and Safari).

### Debugging Log

```Text
execution error: Error: Error: exception raised by object: *** +[NSString stringWithUTF8String:]: NULL cString (-2700)
```

### Workflow Configuration

No change from default settings (just my custom path to markdown file).

### Checklist

- [X] I have [updated to the latest version](./releases/latest) of this workflow.
- [X] I am using Alfred 5. (Older versions are not supported anymore.)

**Comment by chrisgrieser** [comment]
Ah, that's a bug that only occurs when you have an empty reading list. And since my reading list hasn't been empty in a loooong time, I never encountered the problem 🙃 

Fixed in release 1.0.1 https://github.com/chrisgrieser/alfred-read-later/releases/download/1.0.1/alfred-read-later.alfredworkflow

**Comment by cands** [comment]
Thank you! I confirm that it is working now.
